app/models/accounting/transaction.rb

```diff
@@ -22,7 +22,7 @@
 #
 # Indexes
 #
-#  acc_trans_qbid_qbtype__unq_idx                          (qb_id,qb_transaction_type) UNIQUE
+#  acc_trans_qbid_qbtype_unq_idx                           (qb_id,qb_transaction_type) UNIQUE
```

Comment:
Should this change? We've been keeping the double underscore.

Reply:
Crap, sorry. I have no idea. Up to you all.